## ▸ 个人简介

海南比勒费尔德应用科学大学计算机专业副教授。曾任上海交通大学医学院副研究员，合作PI为李令杰研究员。曾于美国辛辛那提儿童医院访学，合作者为Qing Richard Lu教授和Yi Zheng教授。博士毕业于复旦大学生命科学学院计算生物学系，导师为田卫东教授。本科毕业于山东大学数学学院，信息安全专业，本科毕业后经保研直接攻读博士学位。主要研究领域为单细胞与空间多组学算法开发及创新应用，有多年算法开发及数据分析经验。主持国自然青年项目及上海市科委扬帆计划项目，参与多个国家级、省部级项目。主要研究发表在Cell Reports, Cell Discovery，Brief Bioinform等杂志，合作研究发表在Cancer Cell, Cell Stem Cell, PNAS等杂志。


## ▸ 主要研究

---------------------------------------


<p float="left">
<img src="https://fzhang.bioinfo-lab.com/img/tools/logo_fwp.png" height="60">
<img src="https://fzhang.bioinfo-lab.com/img/inferloop_logo.jpg" height="60">
<img src="https://fzhang.bioinfo-lab.com/img/tools/logo_fitdevo.png" height="60">
<img src="https://fzhang.bioinfo-lab.com/img/tools/logo_scmagic.png" height="60">
<img src="https://fzhang.bioinfo-lab.com/img/tools/logo_vector.png" height="60">
<img src="https://fzhang.bioinfo-lab.com/img/tools/logo_beer.png" height="60">
<img src="https://fzhang.bioinfo-lab.com/img/tools/logo_phenopro.png" height="60">
<img src="https://fzhang.bioinfo-lab.com/img/tools/logo_sprint.png" height="60">
</p>

  * 基于基因及调控区进化保守性评估细胞和组织发育潜能的定量分析, **上海交通大学学报(医学版)**, 2023, [论文](https://xuebao.shsmu.edu.cn/CN/10.3969/j.issn.1674-8115.2023.11.006), 共同通讯作者

  * Feature-weight based measurement of cancerous transcriptome using cohort-wide and sample-specific information, **Cellular Oncology**, 2023, {[代码](https://github.com/jumphone/FWP),  [论文](https://doi.org/10.1007/s13402-023-00879-6), [免费全文](https://rdcu.be/dn9I1)}, 通讯作者

  * InferLoop: leveraging single-cell chromatin accessibility for the signal of chromatin loop, **Briefings in Bioinformatics**, 2023, {[代码](https://github.com/jumphone/InferLoop), [论文](https://doi.org/10.1093/bib/bbad166)}, 第一&共同通讯作者
    
  * FitDevo: accurate inference of single-cell developmental potential using sample-specific gene weight, **Briefings in Bioinformatics**, 2022, {[代码](https://github.com/jumphone/FitDevo), [论文](https://doi.org/10.1093/bib/bbac293)}, 第一&共同通讯作者
  
  * scMAGIC: accurately annotating single cells using two rounds of reference-based classification, **Nucleic Acids Research**, 2022, {[代码](https://github.com/TianLab-Bioinfo/scMAGIC), [论文](https://doi.org/10.1093/nar/gkab1275)}, 共一作者
    
  * Unsupervised Inference of Developmental Directions for Single Cells Using VECTOR, **Cell Reports**, 2020, {[代码](https://github.com/jumphone/Vector), [论文](https://doi.org/10.1016/j.celrep.2020.108069)}, 第一作者

  * A novel approach to remove the batch effect of single-cell data, **Cell Discovery**, 2019, {[代码](https://github.com/jumphone/BEER), [论文](https://doi.org/10.1038/s41421-019-0114-x)}, 第一作者
    
  * PhenoPro: a novel toolkit for assisting in the diagnosis of Mendelian disease, **Bioinformatics**, 2019, {[代码](https://github.com/jumphone/PhenoPro), [论文](https://doi.org/10.1093/bioinformatics/btz100)}, 共一作者

  * SPRINT: an SNP-free toolkit for identifying RNA editing sites, **Bioinformatics**, 2017, {[代码](https://github.com/jumphone/SPRINT), [论文](https://doi.org/10.1093/bioinformatics/btx473)}, 第一作者

    
    
    
<img src="https://fzhang.bioinfo-lab.com/img/white.png" height="50">

---------------------------------------

## ▸ 合作研究

---------------------------------------

<img src="https://fzhang.bioinfo-lab.com/img/colla_logo.png" width="200" height="50">

  * Ultrasmall Cu2I2 nanoclusters trigger metabolic-epigenetic reprogramming and endogenous antioxidant systems for alleviating osteoarthritis, **Chemical Engineering Journal**, 2024, [论文](https://doi.org/10.1016/j.cej.2024.154568)
  
  * ARID1A loss promotes RNA editing of CDK13 in an ADAR1-dependent manner, **BMC Biology**, 2024, [论文](https://doi.org/10.1186/s12915-024-01927-9)

  * Long-read sequencing reveals alternative splicing-driven, shared immunogenic neoepitopes regardless SF3B1 status in uveal melanoma, **Cancer Immunology Research**, 2023, [论文](https://doi.org/10.1158/2326-6066.cir-23-0083)
  
  * Kinase-independent role of mTOR and on-/off-target effects of an mTOR kinase inhibitor, **Leukemia**, 2023, [论文](https://doi.org/10.1038/s41375-023-01987-w)
  
  * Loss of phosphatase CTDNEP1 potentiates aggressive medulloblastoma by triggering MYC amplification and genomic instability, **Nature Communications**, 2023, [论文](https://doi.org/10.1038/s41467-023-36400-8)
  
  * Inducible lncRNA transgenic mice reveal continual role of HOTAIR in promoting breast cancer metastasis, **Elife**, 2022, [论文](https://doi.org/10.7554/elife.79126)
  
  * Single-cell multiomics identifies clinically relevant mesenchymal stem-like cells and key regulators for MPNST malignancy, **Science Advances**, 2022, [论文](https://doi.org/10.1126/sciadv.abo5442)
  
  * Conserved and Distinct Functions of the Autism-Related Chromatin Remodeler CHD8 in Embryonic and Adult Forebrain Neurogenesis, **Journal of Neuroscience**, 2022, [论文](https://doi.org/10.1523/JNEUROSCI.2400-21.2022)
  
  * CDC42 controlled apical-basal polarity regulates intestinal stem cell to transit amplifying cell fate transition via YAP-EGF-mTOR signaling, **Cell Reports**, 2022, [论文](https://doi.org/10.1016/j.celrep.2021.110009)

  * OLIG2 maintenance is not essential for diffuse intrinsic pontine glioma cell line growth but regulates tumor phenotypes, **Neuro-Oncology**, 2021, [论文](https://doi.org/10.1093/neuonc/noab016)

  * Single-cell transcriptomic analysis reveals a hepatic stellate cell-activation roadmap and myofibroblast origin during liver fibrosis, **Hepatology**, 2021, [论文](https://doi.org/10.1002/hep.31987)

  * Adaptive responses to mTOR gene targeting in hematopoietic stem cells reveal a proliferative mechanism evasive to mTOR inhibition, **Proceedings of the National Academy of Sciences of the United States of America**, 2020, [论文](https://www.pnas.org/content/118/1/e2020102118)
 
  * CTCF-mediated chromatin looping in EGR2 regulation and SUZ12 recruitment critical for peripheral myelination and repair, **Nature Communications**, 2020, [论文](https://doi.org/10.1038/s41467-020-17955-2)

  * The CAFA challenge reports improved protein function prediction and new functional annotations for hundreds of genes through experimental screens, **Genome Biology**, 2019, [论文](https://doi.org/10.1186/s13059-019-1835-8)

  * Single-Cell Transcriptomics in Medulloblastoma Reveals Tumor-Initiating Progenitors and Oncogenic Cascades during Tumorigenesis and Relapse, **Cancer Cell**, 2019, [论文](https://doi.org/10.1016/j.ccell.2019.07.009)

  * Single-cell Transcriptomics Uncovers Glial Progenitor Diversity and Cell Fate Determinants during Development and Gliomagenesis, **Cell Stem Cell**, 2019, [论文](https://doi.org/10.1016/j.stem.2019.03.006)

  * Dual requirement of CHD8 for chromatin landscape establishment and histone methyltransferase recruitment to promote CNS myelination and repair, **Developmental Cell**, 2018, [论文](https://doi.org/10.1016/j.devcel.2018.05.022)

  * Genome-wide identification of regulatory sequences undergoing accelerated evolution in the human genome, **Molecular Biology and Evolution**, 2016, [论文](https://doi.org/10.1093/molbev/msw128)


<img src="https://fzhang.bioinfo-lab.com/img/white.png" height="50">

---------------------------------------

## ▸ 工作经历 & 教育经历

---------------------------------------

&nbsp;&nbsp;&nbsp;&nbsp;

* **副教授**, 计算机科学，海南比勒费尔德应用科学大学, 中国, **2025.02-至今**

* **副研究员**, 组织胚胎学与遗传发育学系, 上海交通大学医学院, 中国, **2024.01-2025.01**

* **助理研究员**, 组织胚胎学与遗传发育学系, 上海交通大学医学院, 中国, **2020.10-2023.12**

* **访问学者**, 辛辛那提儿童医院医学中心, 美国, **2018.01-2020.01**. 合作者: [Prof. Qing Richard Lu](https://www.cincinnatichildrens.org/bio/l/qing-richard-lu) 和 [Prof. Yi Zheng](https://www.cincinnatichildrens.org/bio/z/yi-zheng)

* **博士**, 生物统计学, 生命科学学院, 复旦大学, 中国, **2015.09-2020.06**. 导师: [田卫东 教授](https://life.fudan.edu.cn/9f/60/c31283a368480/page.htm)

* **暑期学校**, 加州大学洛杉矶分校, 美国, **2013.08-2013.09**. 生物统计学导论. 主讲: Prof. Gjertson, D.W.

* **本科**, 数学学院, 山东大学, 中国, **2011.09-2015.06** 

<img src="https://fzhang.bioinfo-lab.com/img/white.png" height="50">

---------------------------------------

## ▸ 联系方式

---------------------------------------

&nbsp;&nbsp;&nbsp;&nbsp;<img src="https://fzhang.bioinfo-lab.com/img/email_logo.png" width="200">

* Comming Soon


